src/protocol/OpenConnectionReply1.php

```diff
@@ -16,17 +16,25 @@
 
 namespace raklib\protocol;
 
+use function is_int;
+
 class OpenConnectionReply1 extends OfflineMessage{
 	public static $ID = MessageIdentifiers::ID_OPEN_CONNECTION_REPLY_1;
 
 	public int $serverID;
-	public bool $serverSecurity = false;
+	public bool $serverSecurity;
```

Comment:
Do we still need a separate `serverSecurity` field if it is already implied by `cookie !== null`?

Reply:
Not really, but the user needs to know what field it is.

Reply:
The user doesn't need. The user can simply check `$packet->cookie !== null` if they want to know.